src/index.js

```diff
@@ -34,6 +135,7 @@ module.exports = function createService(url, options) {
         return this.channel
```

Comment:
Maybe we need to check availability of rabbit and channel somehow, to ensure we are not in "reconnect" mode. WDYT?

Reply:
And the same for sendToQueue

Reply:
I thought about that, probably we should have some mechanism to re-send all of this all messages which are failed to send when reconnection

Reply:
Maybe, it can be done separately then.

Reply:
Yea, need to think about how to deliver delayed response